localesetup/localesetup

```diff
@@ -49,9 +52,9 @@ else
 # start of interactive section
 
 # find current locale
-currentlocale="$(cat /etc/profile.d/lang.sh \
-  | grep -E "^[[:space:]]*export LANG=" \
-  | sed -e "s/^[ ]*export LANG=\(.*\)$/\1/")"
+currentlocale=$(sed -n '
+	 /^ *export LANG=/{
+  	s/^ *export LANG=\(.*\)$/\1/p}' /etc/profile.d/lang.sh)
```

Comment:
There's a leading tab in this line. Just for the sake of uniformity with the rest of the file I think it should better be removed.